ros2doctor/ros2doctor/verb/call.py

```diff
@@ -0,0 +1,178 @@
+# Copyright 2019 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import socket
+import struct
+import threading
+import time
+
+import rclpy
+from rclpy.executors import MultiThreadedExecutor
+from rclpy.node import Node
+from ros2doctor.verb import VerbExtension
+
+from std_msgs.msg import String
+
+DEFAULT_GROUP = '225.0.0.1'
+DEFAULT_PORT = 49150
+DEFAULT_TOPIC = '/knockknock'
+summary_table = {}
+
+
+class CallVerb(VerbExtension):
+    """Publish and subscribe, multicast send and receive, print table."""
+
+    def main(self, *, args):
+        rclpy.init()
+        pub_node = Talker()
+        sub_node = Listener()
+
+        executor = MultiThreadedExecutor()
+        executor.add_node(pub_node)
+        executor.add_node(sub_node)
+        try:
+            count = 0
+            _spawn_summary_table()
+            while True:
+                if (count % 20 == 0 and count != 0):
```

Comment:
Rather than counting to a number, I think it is more intuitive to run the loop at a given period (e.g. in seconds). I would make this period configurable with a command line option.